captcha/events.py

```diff
@@ -18,7 +20,12 @@ class Listeners(MixinMeta, metaclass=ABCMeta):
     async def runner(self, member: Member):
         allowed = await self.basic_check(member)
         if allowed:
-            challenge = await self.create_challenge_for(member)
+            try:
+                challenge = await self.create_challenge_for(member)
+            except AlreadyHaveCaptchaError:
+                raise AlreadyHaveCaptchaError(
+                    "The user already have a captcha object running."
+                )
```

Comment:
```suggestion
            challenge = await self.create_challenge_for(member)
```
This seems pointless to me, re-raising the same error. We can just allow it to raise like beforehand